doc/doxygen/src/creating-modules.md

```diff
@@ -67,8 +67,13 @@ their dependencies.
 Modules outside of RIOTBASE                      {#modules-outside-of-riotbase}
 ===========================
 Modules can be defined outside `RIOTBASE`. In addition to add it to `USEMODULE`
-the user needs to add the path to the module to `EXTERNAL_MODULE_DIRS` and add
-the include path to the API definitions to `INCLUDES`.
+the user needs to add the path to the module to `EXTERNAL_MODULE_DIRS`.
+The external module can define a `Makefile.include` file to add the include path
```

Comment:
This is misleading."Makefile.include" will always be included by every submake, thus allows global definitions. That just happens to also be useful for changing the include path.

Reply:
It is not included by every submake, just by the main `Makefile.include`.

But yeah it could be used for more than just setting `USEMODULE_INCLUDES`, I could add a note on setting CFLAGS for example.

Reply:
> It is not included by every submake

Right. But any variable that is exported will be available in every submake.

Reply:
I will reformulate in this direction.